cli/pkg/providers/gcp/kubeConfigTemplate.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+clusters:
+- cluster:
+    server: {{.EndpointUrl}}
+    certificate-authority-data: {{.CertificateData}}
+  name: {{.ClusterName}}
+contexts:
+- context:
+    cluster: {{.ClusterName}}
+    user: {{.ClusterName}}
+  name: {{.ClusterName}}
+current-context: {{.ClusterName}}
+kind: Config
+preferences: {}
+users:
+- name: {{.ClusterName}}
```

Comment:
Is this file used? I think we can remove it from Bazel, as Terraform generates the kubeconfig file.